nixops_aws/backends/ec2.py

```diff
@@ -1666,32 +1663,19 @@ def create(self, defn, check, allow_reboot, allow_recreate):
                     self._conn, volume.id, self.logger
                 )
 
-        # Always apply tags to the volumes we just created.
+        # Always apply tags to the volumes created
         for device_stored, v in self.block_device_mapping.items():
             device_real = device_name_stored_to_real(device_stored)
 
-            if not (
```

Comment:
This has been reverted per https://github.com/NixOS/nixops-aws/pull/102/commits/45849248b96048031c686b6f449b836ecc765b45